tests/test_integration/test_eval_loop_integration.py

```
@@ -801,6 +838,77 @@ async def verify(self, response, expected, **meta):
+
+    async def close(self):
+        pass
+
+
+class TestVerifyOnlyResumeReexecution:
+    """FEP-888 regression: a synthetic checkpoint mirroring Run A (real-schema, produced by
+    run_evaluation). On a kill that lands after the solve checkpoint but before verification,
+    verify-only sandbox rollouts with a recoverable/successful solve must re-execute, while a
+    genuine solve failure must replay graded-0 without a second attempt."""
+
+    def _patch_lifecycle(self, monkeypatch):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_patch_lifecycle`

Add return type annotation: `None`

(ANN202)